tests/test_cli.py

```diff
@@ -697,3 +738,23 @@ def test_CustomJsonEncoder_with_non_json_serialisable_object(self):
         encoder = sceptre.cli.CustomJsonEncoder()
         response = encoder.encode(datetime.datetime(2016, 5, 3))
         assert response == '"2016-05-03 00:00:00"'
+
+    def test_remove_metadata(self):
+        original_response = {
+            "Parameters": {
+                "Key 1": "Value 1",
```

Comment:
remove extra content from this response not useful in test - could just have ` {"Key" : "Value", "ResponseMetadata": etc}`

Reply:
Also if you remove the _remove_response_metadata function - no need for this test.